DeepMRSeg/deepmrseg_test.py

```diff
@@ -38,63 +43,57 @@ def read_flags():
 	# Only parse for "--config"
 	configarg,_ = parser.parse_known_args()
 
-#	INPUT LIST
+#	I/O
 #	==========
-	inputArgs = parser.add_argument_group( 'REQUIRED INPUT ARGS' )
+	inputArgs = parser.add_argument_group( 'INPUT SCANS - REQUIRED', 'Select one of the 3 options: Single case (inImg/outImg), process all scans in a folder (inDir/outDir/outSuff) or using a scan list (sList)')
```

Comment:
Print help in multiple lines rather than a long line. For e.g.:

'Select one of the 3 options: \
     I/O Option 1: Single case (inImg/outImg), \
     I/O Option 2: process all scans in a folder (inDir/outDir/outSuff) or \
     I/O Option 3: using a scan list (sList)'